src/app/pages/listings/listings.component.spec.ts

```diff
@@ -11,7 +11,7 @@ describe('ListingsComponent', () => {
       imports: [ListingsComponent]
     })
     .compileComponents();
-    
+    console.log('testing listing component.');
```

Comment:
Please use ternary operator instead of if else

Reply:
Thank you for noticing this.